### PR TITLE
Added reaction support to projectBytId

### PR DESCRIPTION
**Pull request by aminlatifi**
Fixed projectBySlug endpoint parameter types

### CODE REVIEW

File: src/resolvers/projectResolver.test.ts [hunk 1]
```diff
@@ -1830,6 +1833,81 @@ function likedProjectsByUserIdTestCases() {
...
+    const result = await axios.post(graphqlUrl, {
+      query: projectByIdQuery,
+      variables: {
+        id: project.id,
+        connectedWalletUserId: user.id,
+      },
+    });
+    assert.equal(result.data.data.projectById.id, project.id);
+    assert.equal(result.data.data.projectById.reaction.id, reaction.id);
+  });
```
Comment (mohammadranjbarz) [inline src/resolvers/projectResolver.test.ts]: @aminlatifi  I wrote some test cases for `projectById` query and this test fails, I don't know it's my test case problem or your code

error message
```
       should return reaction when user liked the project:
     QueryFailedError: null value in column "reaction" of relation "reaction" violates not-null constraint
```

Comment (aminlatifi) [reply src/resolvers/projectResolver.test.ts]: Thanks for making it complete by a test. The reaction creation on test had an issue not the graphql endpoint. It's fixed by this
https://github.com/Giveth/impact-graph/pull/358/commits/e979c95e2c3a07a1524abdb22833e5c7dee2c98d

Comment (mohammadranjbarz) [reply src/resolvers/projectResolver.test.ts]: Great, thank you man

**Comment by mohammadranjbarz** [comment]
@aminlatifi  LGTM, thanks.

**Pull request merged**